Tests/MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -503,31 +503,45 @@ class NavigationServiceTests: XCTestCase {
         let now = Date()
         let trace = Fixture.generateTrace(for: route).shiftedToPresent()
 
-        trace.forEach { navigation.router.locationManager!(navigation.locationManager, didUpdateLocations: [$0]) }
+        trace.forEach {
+            navigation.router.locationManager!(navigation.locationManager, didUpdateLocations: [$0])
```

Comment:
Sorry for being too peeky regarding forced unwrapping :) I guess we can just omit it here?

Reply:
Yep, will change